### PR TITLE
pull in upstream changes

### PR DESCRIPTION
**Pull request by pathouse**

### CODE REVIEW

File: security_groups.tf [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
   }
 }
 
 resource "aws_security_group_rule" "redis_ingress" {
-  count                    = "${length(var.allowed_security_groups)}"
+  count                    = length(var.allowed_security_groups)
   type                     = "ingress"
-  from_port                = "${var.redis_port}"
-  to_port                  = "${var.redis_port}"
+  from_port                = var.redis_port
```
Comment (nimitzshah) [inline security_groups.tf:14]: Is this because we upgraded terraform that we dont need strings?

Comment (pathouse) [reply security_groups.tf:14]: i'm actually shutting this down and pulling this module entirely into our main redis module

becuase

1) this module is very small and hasn't been changed in ~a year so its stable
2) it uses its own naming conventions that conflict with our own and makes it impossible to import existing production resources into this config w/out destroying and recreating them which i do not want to do

Comment (nimitzshah) [reply security_groups.tf:14]: 🎉  for less repos!!

Comment (pathouse) [reply security_groups.tf:14]: haha yes exactly
https://github.com/DogParkLabs/terraform-aws-redis/pull/8

**Pull request closed without merging**